doozer/doozerlib/cli/images.py

```diff
@@ -376,6 +376,105 @@ def dgr_rebase(image_meta, terminate_event):
 
 
+@cli.command("k:images:rebase", short_help="Refresh a group's distgit content from source content.")
+@click.option("--version", metavar='VERSION', default=None, callback=validate_semver_major_minor_patch,
+              help="Version string to populate in Dockerfiles. \"auto\" gets version from atomic-openshift RPM")
+@click.option("--release", metavar='RELEASE', default=None, help="Release string to populate in Dockerfiles.")
+@click.option("--embargoed", is_flag=True, help="Add .p1 to the release string for all images, which indicates those images have embargoed fixes")
+@click.option("--repo-type", metavar="REPO_TYPE", envvar="OIT_IMAGES_REPO_TYPE",
+              default="unsigned",
+              help="Repo group type to use for version autodetection scan (e.g. signed, unsigned).")
+@click.option("--force-yum-updates", is_flag=True, default=False,
+              help="Inject \"yum update -y\" in the final stage of an image build. This ensures the component image will be able to override RPMs it is inheriting from its parent image using RPMs in the rebuild plashet.")
+@option_commit_message
+@option_push
+@pass_runtime
+def k_images_rebase(runtime: Runtime, version: Optional[str], release: Optional[str], embargoed: bool, repo_type: str, force_yum_updates: bool, message: str, push: bool):
+    runtime.initialize(clone_distgits=False)
+
+    if runtime.group_config.public_upstreams and (release is None or release != "+" and not release.endswith(".p?")):
+        raise click.BadParameter("You must explicitly specify a `release` ending with `.p?` (or '+') when there is a public upstream mapping in ocp-build-data.")
+
+    # This is ok to run if automation is frozen as long as you are not pushing
+    if push:
+        runtime.assert_mutation_is_permitted()
+
+    cmd = runtime.command
+    runtime.state[cmd] = dict(state.TEMPLATE_IMAGE)
+    lstate = runtime.state[cmd]  # get local convenience copy
+
+    # If not pushing, do not clean up our work
+    runtime.remove_tmp_working_dir = push
+
+    # Get the version from the atomic-openshift package in the RPM repo
+    if version == "auto":
+        version = runtime.auto_version(repo_type)
+
+    if version and not runtime.valid_version(version):
+        raise ValueError(
+            "invalid version string: {}, expecting like v3.4 or v1.2.3".format(version)
+        )
+
+    metas = runtime.ordered_image_metas()
+    lstate['total'] = len(metas)
+
+    def dgr_rebase(image_meta, terminate_event):
+        try:
+            k_dgr = KonfluxImageDistGitRepo(image_meta)
+            if embargoed:
+                k_dgr.private_fix = True
+            (real_version, real_release) = k_dgr.rebase_dir(version, release, terminate_event, force_yum_updates)
+            sha = k_dgr.commit(message, log_diff=True)
+            k_dgr.tag(real_version, real_release)
+            runtime.add_record(
+                "distgit_commit",
```

Comment:
differentiate database records so that they cannot be mistaken for standard pipeline.
```suggestion
                "k_distgit_commit",
```

Reply:
added in https://github.com/openshift-eng/art-tools/pull/543/commits/20ff4db345bfc9f5275e44da5753ae9d7cec7e5c